### PR TITLE
Add Debian 9 as an optional queue which can be requested specifically

### PR DESCRIPTION
**Pull request by smile21prc**
Add extra "," at the end of queues to make diff tools happiler.

### CODE REVIEW

**Comment by smile21prc** [comment]
@dotnet-bot test ci please

**Comment by smile21prc** [comment]
Thanks @danmosemsft .

**Comment by smile21prc** [comment]
@dotnet-bot test ci please

File: buildpipeline/portable-linux.groovy [hunk 1]
```diff
@@ -63,8 +63,10 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
                                      'Ubuntu.1610.Amd64.Open',
                                      // 'Ubuntu.1704.Amd64.Open',
                                      'suse.422.amd64.Open',
-                                     'fedora.25.amd64.Open']
-
+                                     'fedora.25.amd64.Open',]
+            if (params.OuterLoop) {
```
Comment (mmitche) [inline buildpipeline/portable-linux.groovy]: LGTM

**Comment by smile21prc** [comment]
@dotnet-bot test Portable Windows x64 Debug Build

**Comment by mmitche** [comment]
@smile21prc You don't need test ci please unless you're changing the pipelinejobs.groovy file
running outerloop should show the new runs

@dotnet-bot test Portable Outerloop Windows x64 Release Build

**Comment by mmitche** [comment]
@dotnet-bot test Portable Outerloop Windows x64 Debug Build

**Comment by smile21prc** [comment]
@dotnet-bot test Portable Windows x64 Release Build

File: buildpipeline/portable-linux.groovy (hunk 1 above)
Comment (smile21prc) [inline buildpipeline/portable-linux.groovy]: Thanks @mmitche

**Comment by smile21prc** [comment]
@mmitche , got that, thanks for the info.

**Comment by smile21prc** [comment]
@dotnet-bot test Portable Outerloop Windows x64 Release Build

**Comment by smile21prc** [comment]
Thanks @weshaggard for the review.

**Pull request merged**